makefiles/boards/stm32.inc.mk

```diff
@@ -1,5 +1,5 @@
 PROGRAMMER ?= openocd
-PROGRAMMERS_SUPPORTED += bmp dfu-util openocd stm32flash
+PROGRAMMERS_SUPPORTED += bmp dfu-util openocd stm32flash cpy2remed
```

Comment:
This is not supported by all stm32 boards, so far only `nucleo-l552ze-q` provides the needed configuration

Reply:
Unfortunately yes. I do not have access to other Nucleo boards and I cannot check this. Moreover, I suspect that each board have different removable media name (directory in Linux). Due to this fact I introduce DIR_NAME_AT_REMED variable.

Reply:
Yea so let's just add this for `nucleo-l552ze-q` for now.